src/overlays/actors/ovl_Obj_Tokeidai/z_obj_tokeidai.c

```diff
@@ -843,8 +843,8 @@ void ObjTokeidai_Clock_Draw(Actor* thisx, GlobalContext* globalCtx) {
         gSPDisplayList(POLY_OPA_DISP++, gClockTowerClockFace);
     }
 
-    Matrix_InsertTranslation(0.0f, -1112.0f, -19.6f, MTXMODE_APPLY);
-    Matrix_RotateY((s16)this->sunMoonPanelRotation, MTXMODE_APPLY);
+    Matrix_Translate(0.0f, -1112.0f, -19.6f, MTXMODE_APPLY);
+    Matrix_RotateYS((s16)this->sunMoonPanelRotation, MTXMODE_APPLY);
```

Comment:
```suggestion
    Matrix_RotateYS(this->sunMoonPanelRotation, MTXMODE_APPLY);
```
Just checked, shouldn't be needed.